package.json

```diff
@@ -56,6 +56,7 @@
     "babel-runtime": "^6.26.0",
     "iopipe": "^1.2.0",
     "lodash.get": "^4.4.2",
+    "node-gyp": "^3.6.2",
```

Comment:
I think this belongs in the v8-profiler-lambda package?

https://github.com/iopipe/v8-profiler-lambda/blob/master/package.json

Reply:
+1

Reply:
@ewindisch https://github.com/iopipe/v8-profiler-lambda/pull/1